src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -526,10 +526,6 @@ int32 AuraEffect::CalculateAmount(Unit* caster)
         switch (GetAuraType())
         {
             case SPELL_AURA_PERIODIC_DAMAGE:
-                // xinef: save caster depending auras, always pass 1 as stack amount, effect will be multiplicated at the end of the function by correct value!
```

Comment:
why?

Reply:
It is the same code as case SPELL_AURA_PERIODIC_LEECH. This case slips into case SPELL_AURA_PERIODIC_LEECH now. :)

Reply:
ok agree, just ask because of the second case. to make sure it's okay.

Reply:
this can be resolved